### ISSUE TITLE
GH deconstructors to output list where needed

### ISSUE THREAD

**Issue by samnyb** [strusoft/femdesign-api#198]
![image](https://user-images.githubusercontent.com/97892858/153435155-6aca3909-9891-4694-8859-9bd390d36f5f.png)

When deconstructing beam elements, the data is initially all in one list, but Crossections, Connectivity and Eccentrity are all two-per-item, so it branches out and becomes two items per initial beam. 

When reassembling using `Bars.Beam`, the resulting data structure is all out of order, with a data tree instead of the initial list. What's more, with a complete set of beams on every branch, the data have grown to n^2 items in total.

**I have only tested this on beams so far**, but presumably there might be similar problems on any deconstruct/create pair where the data branches. An example file for this issue has been added to a Test files folder.

**Comment by xRadne** [comment]
### General problem
Taking the `LoadCombination.Deconstruct` component as an example:

The expected output would be for each input lcomb. to have a list of data for each output param. 
Since each lcomb. contains a list of LoadCases.

Currently the output needs to be grafted. But I don't think many users would expect this to just deconstruct-reconstruct on object.

![image](https://user-images.githubusercontent.com/32938476/153852677-737805d4-bf55-4175-8fa2-f827d8981939.png)

I suggest the following lines are changed to all have `GH_ParamAccess.list`. This would solve the problem.
https://github.com/strusoft/femdesign-api/blob/3e2c1fb3dc5cf725afa175c7544813c3c8f5d253/FemDesign.Grasshopper/Deconstruct/LoadCombinationDeconstruct.cs#L17-L24

### Back to beams
The same could _and should_ be done for **all deconstructors** that have **multiple outputs per element**. 

This is how David Rutten have designed the grasshopper tree-paths-datastructure.

**Comment by Marco-Pellegrino** [comment]
@samnyb @xRadne

I did totally agree with both of you and I have modified the components output to allow an easier construct and deconstruct workflow.
I have modified the following component:

https://github.com/strusoft/femdesign-api/blob/db5080306d7eb518a7b28776c49d662e328bc103/FemDesign.Grasshopper/Deconstruct/BarsDeconstruct.cs#L54-L82

https://github.com/strusoft/femdesign-api/blob/db5080306d7eb518a7b28776c49d662e328bc103/FemDesign.Grasshopper/Deconstruct/LoadCombinationDeconstruct.cs#L42-L57


It looks like that changing `GH_ParamAccess.item` to `GH_ParamAccess.list` wasn't enough for Grasshopper to interpret the output as a list/datatree. I am not sure if there is a better way but it seems to work and it gives what we were expecting.

```csharp
pManager.AddTextParameter("Guid", "Guid", "Guid.", GH_ParamAccess.list);
```

![image](https://user-images.githubusercontent.com/53620289/161249870-3c73e801-c6b1-4306-bf37-d53e5c81d3da.png)
